apps/aechannel/test/aesc_fsm_SUITE.erl

```diff
@@ -1224,6 +1238,21 @@ shutdown_(#{fsm := FsmI, channel_id := ChannelId} = I, R) ->
     verify_close_mutual_tx(SignedTx, ChannelId),
     ok.
 
+settle_(TTL, MinDepth, #{fsm := FsmI, channel_id := ChannelId} = I, R, Debug) ->
+    ok = rpc(dev1, aesc_fsm, settle, [FsmI]),
+    _ = await_signing_request(settle_tx, I),
+    ct:log("settle_tx signed", []),
+    mine_blocks(dev1, TTL + 1, Debug),
+    SignedTx = await_on_chain_report(I, #{info => channel_closed}, ?TIMEOUT),
```

Comment:
How many confirmations are waited for before reporting this?

And if transaction gets included on chain, and then excluded from chain, does a new report get sent?

And if transaction gets included on chain, then excluded, then included again, are a total of 3 reports sent? (on, off, on)